__tests__/component/form/CreateCustomerForm.test.tsx

```diff
@@ -50,15 +51,30 @@ const fillAndSubmitForm = async (customer: Customer) => {
     await waitFor(() => fireEvent.click(submitButton))
 }
 
+const mockRequest = (): object => bodySpy.getCalls()[0].args[0]
+
+const checkRequired = (
```

Comment:
Please create a `util.ts` file in the folder `__test__/` and export this function from there. You could also rename this to `checkTestRequestBodyRequiredFields`.